### ISSUE TITLE
Losse do not fall and AP is 0

### ISSUE THREAD

**Issue by Luhui-Zhao** [IDEA-opensource/DAB-DETR#73]
Hello, I am using your DAB-DETR training data set cityscapes. The loss at the beginning is 20. As the training goes on, the loss decreases to about 14 and no longer decreases, and the test result is also 0. Logically speaking, the loss value should continue to decline. I adjusted the learning rate and random seed parameters, but the result remained unchanged. Could someone please tell me what to do? Thank you very much.
